cpu/esp32/esp-eth/Kconfig

```diff
@@ -0,0 +1,24 @@
+# Copyright (c) 2022 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+config MODULE_ESP_ETH
```

Comment:
@gschorcht I could not find a dependency on a feature like `periph_eth` in the `Makefile.dep`, but I observe compilation errors on boards that do not provide a configuration. Should this module depend on `periph_eth` (which would be only provided when a config is present)?

Reply:
The board defines it as feature here: https://github.com/RIOT-OS/RIOT/blob/bec2868223b3559bdd969de5cd090a0c279baea5/boards/esp32-olimex-evb/Makefile.features#L15 Maybe this is wrong.